css/main.css

```diff
@@ -1,98 +1,826 @@
+  height: 1.875rem;
+  margin-right: 0.625rem;
+}
+
+.signup-section div span img {
+  width: 1.875rem;
+  height: 1.875rem;
+  margin-right: 0.625rem;
+}
+
+.features-articles-content {
+  border: 0.063rem solid #ddd;
+}
+
+.magazine-header {
+  justify-content: center;
+}
+
+.category {
+  margin: 0.625rem 0;
+}
+
+.category .first-element {
+  padding: 0.1875rem 0.4375rem;
+  background-color: #f72210;
+  color: #fff;
+  margin-right: 0.4375rem;
+}
+
+.magazine-section {
+  flex-wrap: wrap;
+}
+
+.magazine-article {
+  width: 32%;
+}
+
+.features-articles h4:hover {
+  background-color: #ccc;
+}
+
+.magazine-bottom {
+  flex-wrap: wrap;
+}
+
+.bottom-magazine-article #bottom-magazine-article-4-title a {
+  margin-top: 0.625rem;
+}
+
+/* featured articles */
+
+.logo-newsweek {
+  width: 1.875rem;
+  height: 1.875rem;
+  margin-right: 0.625rem;
+}
+
+.editors-pick-articles {
+  flex-wrap: wrap;
+}
+
+.featured-slideshows-articles {
+  flex-wrap: wrap;
+}
+
+#rank-number {
+  width: 4.5rem;
+  height: 3.125rem;
+  display: block;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  z-index: 1;
+}
+
+.slide-shows-header h3,
+.editors-pick-header h3 {
+  font-size: 19px;
+  line-height: 30px;
+  color: #333;
+}
+
+/* categories sections */
+
+.main-section-articles {
+  flex-wrap: wrap;
+}
+
+.main-section-articles .featured {
+  width: 32%;
+}
+
+.main-section-header {
+  padding-top: 1em;
+  font-size: 1.375rem;
+  line-height: 1.5rem;
+  color: #f72210;
+}
+
+.featured span a img {
+  height: 16.5625rem;
+}
+
+/* subscriber-newsletter section */
+
+.card-image-subscribe {
+  width: 10.625rem;
+  height: 100%;
+}
+
+.subscribe-heading {
+  font-family: 'Roboto Condensed', sans-serif;
+  font-size: 1rem;
+  line-height: 1rem;
+  color: #f72210;
+  font-weight: 700;
+}
+
+.subscribe-contents div h3 {
+  font-size: 1.5rem;
+  line-height: 1.75rem;
+  font-weight: 700;
+}
+
+.subscribe-contents div p {
+  font-size: 0.935rem;
+  line-height: 1.375rem;
+  font-weight: 400;
+  color: #333;
+}
+
+.btn-subscribe,
+.signup-button {
+  background-color: #f72210;
+  color: #fff;
+}
+
+.subscribe-section {
+  background-color: #f4f5f6;
+}
+
+.newsletter-content {
+  border: 0.063rem solid #ddd;
+}
+
+/* footer section */
+
+.top-footer-section {
+  border-bottom: 0.063rem solid #ddd;
+}
+
+.upper-footer {
+  border-bottom: 0.063rem solid #ddd;
+  width: 100%;
+  margin: auto;
+}
+
+.upper-footer-right {
+  flex-direction: row-reverse;
+}
+
+.menu {
+  list-style: none;
+  display: inline;
+}
+
+.menu li {
+  display: inline-block;
+  padding: 0 0.625rem;
+  border-right: 0.063rem solid #aaa;
+  line-height: 0.935rem;
+}
+
+/* media queries for responsiveness */
+
+/* first breakpoint for tablets 990px */
+
+@media screen and (max-width: 991px) {
+  * {
+    font-size: 90%;
+  }
+
+  .top-container {
+	  width: 100%;
+  }
+
+  #current-date {
+    display: none;
+  }
+
+  .page-header {
+    width: 100%;
+    padding-top: 0.5rem;
+	  padding-bottom: 0.5rem;
```

Comment:
#### `REQUIRED`

Please, fix the indentation. :blush:
A good indentation is fundamental for good readability of the code.
Many Code Editor can help you with this task.
This is VS Code, for example as shown below using [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode):

![Screenshot of how to format code using VS Code](https://res.cloudinary.com/bolaah/image/upload/v1563045186/github-microverse-project/code-reviews/code-review002.gif)